brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1108 InfoBox: Added Letter spacing and line height options for CTA

### Description
- Added Letter spacing and line-height options for CTA
- CTA Text decoration not working at the frontend
- CTA border is affecting if CTA type changes from button to text

### Screenshots
Done - https://share.getcloudapp.com/eDuXg0O2

### Types of changes




Improvement

### How has this been tested?
- Drag drop Infobox
- Select CTA type button/text
- check style -> cta -> typography

### Checklist:
